test/service/staging/test_staging_controller.py

```diff
@@ -92,10 +93,10 @@ def _upload_to_staging(self, data):
             data=data
         )
 
-    def _assert_file_in_staging(self, file_name):
-        file = Path(os.path.join(self.staging_dir, file_name))
+    def _assert_file_in_staging(self, file_path):
+        file = Path(os.path.join(self.staging_dir, file_path))
         if not file.is_file():
-            raise AssertionError(f"File '{file_name}' was not present in the "
+            raise AssertionError(f"File '{file_path}' was not present in the "
```

Comment:
Warum past tense?

Reply:
Weil es zum Zeitpunkt, an dem man die Nachricht liest sicher nicht mehr da ist, weil `tearDown()` es in jedem Fall löscht.